### DISCUSSION TITLE
[QUESTION]: position sticky

### DISCUSSION THREAD

**Discussion by filipSu** [Lodin/react-vtree#31] [Q&A] [answered]
Hi, I'm not sure, if this is a feature request or I just need an example, but I would like to make non-leaf elements 'sticky'. Is there a way? Similar to https://github.com/marchaos/react-virtualized-sticky-tree . Thanks

**Comment by Lodin** [comment, marked as answer]
According to [react-window](https://github.com/bvaughn/react-window#does-this-library-support-sticky-items), it supports sticky items, though it requires some additional code. I haven't try it myself but if the react-window's example is not enough, please let me know: I'll try to prepare some react-vtree specific example.

**Reply by leo-petliakov** [reply]
Hi there
I've been trying to implement sticky positioning from [react-window](https://github.com/bvaughn/react-window#does-this-library-support-sticky-items) documentation for a few days now. 
While I can make nodes stick to top, I can't find a way to implement logic of sticking the parent node to top when its children are being scrolled (and then replacing the sticky node with another one, when a new one's children begin being scrolled). Just like in this [example](https://marchaos.github.io/react-virtualized-sticky-tree/)
Could you please provide an example of doing it, if it's easy for you 
Thanks
